### ISSUE TITLE
2 classes with the same FQDN does not auto-complete

### ISSUE THREAD

**Issue by RobinRadic** [klesun/deep-assoc-completion#153]
Once the plugin finds 2 classes with the same FQDN, it does not auto-complete.
An example:

**composer.json**
```json
    "autoload": {
        "psr-4": {
            "App\\":"app/",
            "Anomaly\\": "overrides/Anomaly",
        },
        "exclude-from-classmap": [
            "vendor/anomaly/streams-platform/src/Ui/Form/FormBuilder.php",
        ]
    }
```
**app/FormBuilderExamples.php**
```php
namespace App;
class FormBuilderExamples {
    public static function options() {
        return [
                'redirect'             => '',
                'permission'           => '',
                'form_view'            => 'streams::form/form',
                'wrapper_view'         => '',
                'layout_view'          => '',
                'breadcrumb'           => '',
                // and a lot more, not needed for this example..
            ];
    }
}
```
**vendor/anomaly/streams-platform/src/Ui/Form/FormBuilder.php**
```php
namespace Anomaly\Streams\Platform\Ui\Form;
class FormBuilder {
    /** @var array = \App\FormBuilderExamples::options() */
    protected $options = [];
}
```
**overrides/Anomaly/Streams/Platform/Ui/Form/FormBuilder.php**
```php
namespace Anomaly\Streams\Platform\Ui\Form;
class FormBuilder {
    /** @var array = \App\FormBuilderExamples::options() */
    protected $options = [];
}
```

**app/MyFormBuilder.php**
```php
namespace App;
class MyFormBuilder extends \Anomaly\Streams\Platform\Ui\Form\FormBuilder{
    protected $options = [
        '<caret>'
    ];
}
```

**Comment by klesun** [comment]
Thanks a lot for so detailed test case. Reproduced, will fix soon.

**Comment by klesun** [comment]
Fixed in [2020.05.11.001](https://plugins.jetbrains.com/plugin/9927-deep-assoc-completion/update/86004)

![image](https://user-images.githubusercontent.com/5202330/81553197-64db8000-938d-11ea-90ee-6baafe906b4e.png)

**Comment by RobinRadic** [comment]
Many, many thanks @klesun !
